app/helpers/ncr/work_orders_helper.rb

```diff
@@ -1,9 +1,8 @@
 module Ncr
   module WorkOrdersHelper
     def approver_options
-      # @todo should this list be limited by client/something else?
       # @todo is there a better order? maybe by current_user's use?
-      User.order(:email_address).pluck(:email_address)
+      User.where(client_slug: 'ncr').order(:email_address).pluck(:email_address)
```

Comment:
Makes sense. Another option would be to create a query object encapsulate this logic and then use the query object in the helper, but this doesn't seem complicated enough to warrant that.

Good blog post on query objects http://blog.codeclimate.com/blog/2012/10/17/7-ways-to-decompose-fat-activerecord-models/